### ISSUE TITLE
Can not change time, add users, etc. Grayed out unlock to change 

### ISSUE THREAD

**Issue by motorcyclerider** [dnschneid/crouton#1497]
I have install Ubuntu 14.04 LTS using crouton on the 2015 Dell Chromebook 11. All went well except I cant change time settings, add users, etc. The "Unlock to change" is greyed out and when clicked it says "Do have permission see admin" Any advice?

**Comment by Watney** [comment]
@motorcyclerider are you seeing the greyed-out settings in Chrome OS or in the chroot (i.e., in Ubuntu).

**Comment by motorcyclerider** [comment]
chroot

**Comment by Watney** [comment]
Pretty strange, @motorcyclerider I'll defer to others. Wish I could offer a *helpful* suggestion

**Comment by motorcyclerider** [comment]
![screenshot from 2015-03-13 23 30 31](https://cloud.githubusercontent.com/assets/11464397/6650128/241ba258-c9d9-11e4-8c87-bb241133b383.png)
![screenshot from 2015-03-13 23 31 24](https://cloud.githubusercontent.com/assets/11464397/6650129/26c73a6c-c9d9-11e4-9ede-fdc1cc5a05fc.png)

**Comment by smibarber** [comment]
tl;dr known issue, can't (reasonably) fix.

polkit rears its ugly head again, see previous discussion at #457. We do have polkit running now, but the policy files (`/usr/share/polkit-1/actions/com.canonical.controlcenter.user-accounts.policy`, `/usr/share/polkit-1/actions/com.canonical.controlcenter.datetime.policy`) only allows users with an active session to authenticate and change the time/add users. For whatever reason, logind isn't considering the session as "active" even though it should be. If you really want, you can go in and change the `no` in the `<allow_any>` tags to `auth_admin_keep`, but I wouldn't recommend it. System time is best left to Chrome OS to manage. Adding another user to login as isn't well tested or supported either, but you're free to try and see what happens.

**Comment by fraserYT** [comment]
Out of interest, I'm just asking the question, before I try it, in case someone knows the answer already and I can save myself a few hours of work.

This isn't an issue that I had before 2 weeks ago and doesn't seem to be an epidemic that's affecting everyone, so while there's no reasonable way to fix it on my existing chroot, and other chroots I've set up seem to inherit the same issue, is it likely to be caused with an issue in my ChromeOS that may be fixed by a complete factory reset? Potentially to an earlier version of ChromeOS?

**Comment by jpw1116** [comment]
@fraserYT if it helps, I went just short of a powerwash: popped out of developer mode then back in again, installing crouton from scratch. The issue persists . . . but I did learn that checking "Weekday" and/or "Date and Month" will nuke the time display from the menu bar. If you can get by with just 12- or 24-hour time and, optionally, seconds then apparently that's as good as it will get for the foreseeable future.
